### PR TITLE
Profile css

### PR DESCRIPTION
**Pull request by Ewiseman**

### CODE REVIEW

**Comment by Ewiseman** [comment]
Hi Eliot, 

What I am attempting to do is have give the "current user"and the "admin role" access to edit the profile.  I've tried many different combinations, including trying to make use of the User Policy (which doesn't work at all).  Nothing seems to be working well.  I either get current user or admin but not both. Any help would be greatly appreciated. 

Thanks!

```
<div class="col-xs-12 col-sm-4 text-center">
      <% if current_user == @user || @user.admin? %>
          <%# if policy(@user).profile_edit? %>
           <%= link_to "Edit Profile Info", edit_user_registration_path, class: "text-center" %> 
       <% end %>  
</div><!--/col-->
```

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -14,18 +13,23 @@ def update
...
+    @staff_users = User.where(role: 'staff')
   end
 
   def show
     @user= User.find(params[:id])
   end
 
+  def edit
+    @user= User.find(params[:id])
+  end
```
Comment (eliotsykes) [inline app/controllers/users_controller.rb]: Any logged in user will be able to view, edit, and update any other users details as there's no authorization checks in this controller.  If that's not acceptable consider adding some authorization checks.

Example scenario:

1. User visits their profile edit page at URL path `/users/7`.
2. User wonders what'll happen if they change the URL to `/users/8`, tries it.

**Comment by eliotsykes** [comment]
Go back to trying to use the policy, that would be the cleaner way of getting this to work, it'll just need a bit of debugging.

In the view:

```html
<% if policy(@user).profile_edit? %>
```

In `UserPolicy#policy_edit?` do some debugging. For example some `puts` statements you can put in the beginning of the `policy_edit?` method:

```ruby
  puts "---------------------"
  # Prove the method that should be called is being called.
  puts "In policy_edit? method"

  # What's the record object in this policy?
  puts "What is this policy's record? '#{record.inspect}'"

  # What's the current/logged-in user object in this policy?
  puts "What is this policy's logged-in/current user? '#{user.inspect}'"

  puts "---------------------"
```

Then try running it. I recommend writing a new feature spec that exercises this functionality too.

**Comment by eliotsykes** [comment]
PS. Those debugging statements will print to the rails server in Terminal.

**Comment by Ewiseman** [comment]
Thanks I will give this a go...

**Pull request merged**